packages/react/src/primitives/types/passwordField.ts

```diff
@@ -9,14 +9,6 @@ export interface PasswordFieldProps extends TextInputFieldProps {
    */
   hideShowPassword?: boolean;
 
-  /**
-   * @description
-   * Set the `aria-label` for hide password button
-   * @default
-   * "Hide password"
-   */
-  hidePasswordButtonLabel?: string;
```

Comment:
Does removing `hidePasswordButtonLabel` and `hidePasswordButtonLabel` constitute a breaking change?

Reply:
Yes, I think we can probably ignore these props, but removing them from the interface is a breaking change since it will cause TS apps to thrown an error. Rather than removing them, @joebuono can you please mark them as `@deprecated` with a message saying they are no longer in use?

Reply:
Hmm good question. I think so?

Reply:
IMO we should preserve the existing behavior to prevent breaking existing usage and add the deprecation warnings, along with supporting the improvements

Reply:
Tagging @hbuchel to get her thoughts when she's back

Reply:
I don't think it's a breaking change to no longer use the `hidePasswordButtonLabel` because it never worked before because it was never announced by the screen reader. With the new `role=switch` there's no reason to include the `hidePasswordButtonLabel` aria label because the state is communicated via the on/off state (see video in pr description). I do think we should leave the prop in the type and simply mark it as being deprecated and unused. 

I think we can wait for @hbuchel's feedback as well.